src/test/java/uk/gov/justice/digital/oasys/controller/OgrsControllerTest.java

```diff
@@ -1,58 +1,33 @@
 package uk.gov.justice.digital.oasys.controller;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
 import io.restassured.RestAssured;
 import io.restassured.config.ObjectMapperConfig;
 import io.restassured.config.RestAssuredConfig;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.boot.web.server.LocalServerPort;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import uk.gov.justice.digital.oasys.api.Ogrs;
 import uk.gov.justice.digital.oasys.jpa.repository.OffenderRepository;
-
 import static io.restassured.RestAssured.given;
 import static org.assertj.core.api.Assertions.assertThat;
 
-
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@RunWith(SpringJUnit4ClassRunner.class)
-@ActiveProfiles("test")
-public class OgrsControllerTest {
-
-    @LocalServerPort
-    int port;
+public class OgrsControllerTest extends IntegrationTest {
 
     @MockBean
     private OffenderRepository offenderRepository;
 
-    @Autowired
-    @Qualifier("globalObjectMapper")
-    private ObjectMapper objectMapper;
-
     @Value("${sample.token}")
     private String validOauthToken;
 
-    @Before
+    @BeforeEach
```

Comment:
Could `@BeforeAll` be used to do the test setup once for the class? Might speed up the tests?